content/end_users/edb.md

```diff
@@ -0,0 +1,7 @@
+---
+title: EDB
+image: logo/edb.svg
+homepage: https://www.enterprisedb.com/
+---
+
+EDB's DataBase as a Service solution relies on CloudNativePG to run PostgreSQL and Postgres Distributed workloads. EDB is one of the primary contributors to the open source PostgreSQL project and the founder of CloudNativePG.
```

Comment:
wait which DBAAS from EDB? I see you've deleted the BigAnimal entry.
Perhaps more specific https://www.enterprisedb.com/products/edb-postgres-ai-cloud-service
?